### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
- Added some styling on the CSS file
- Added mobile version menu functionality
- Added dynamics HTML With JavaScript
- Modified CSS file to make the navbar appear

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -6,32 +6,20 @@ env:
...
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Webhint
+      - name: Setup ESLint
         run: |
-          npm install --save-dev hint@6.x
-          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
-      - name: Webhint Report
-        run: npx hint --telemetry=off .
```
Comment (hemantso) [inline .github/workflows/linters.yml]: - [x] There should not be any custom changes in linter files. Please follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to correctly step linter.

File: index.html [hunk 2]
```diff
@@ -8,16 +8,35 @@
...
+        <div class="div-menu">
+            <div class="logo">
+                <a  href="#">Carlos Feliz</a>
+            </div>
+            <div class="btn">
+                <span >&#9776</span>
+            </div>
+            <ul class="menu" id="menu">
+                <li class="close">
+                    <a href="#" id="close"> X </a>
```
Comment (hemantso) [inline index.html]: - [x] On the desktop view the hamburger menu links are visible. Kindly try to hide it on desktop view.

image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @xJokeer and @Hamzaoutdoors, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Try to add some images of the design in your `README`  file.

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**